sdk/azcore/poller.go

```diff
@@ -200,13 +202,13 @@ func (l *LROPoller) FinalResponse(ctx context.Context, respType interface{}) (*h
 // into the respType interface that is provided.
 func (l *LROPoller) PollUntilDone(ctx context.Context, freq time.Duration, respType interface{}) (*http.Response, error) {
 	logPollUntilDoneExit := func(v interface{}) {
-		Log().Writef(LogLongRunningOperation, "END PollUntilDone() for %T: %v", l.lro, v)
+		logger.Log().Writef(logger.LogLongRunningOperation, "END PollUntilDone() for %T: %v", l.lro, v)
```

Comment:
I think these also should reference the `azcore` instances, just feels more correct. 
```suggestion
		logger.Log().Writef(azcore.LogLongRunningOperation, "END PollUntilDone() for %T: %v", l.lro, v)
```

Reply:
Same comment as above.